app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/details/PaymentDetailExpandableCard.kt

```diff
@@ -54,16 +58,24 @@ internal fun PaymentDetailExpandableCard(
 ) {
   val dateTimeFormatter = rememberHedvigMonthDateTimeFormatter()
   HedvigCard(
-    onClick = onClick,
+    onClick = null,
```

Comment:
Maybe can just remove this one completely? I think we have a HedvigCard without a clickable lambda at all?